docs/requests-packages.md

```diff
@@ -2,15 +2,32 @@ Additional R, Stata, or Python packages in OpenSAFELY must be added by the OpenS
 Installing packages via commands that pull the package from the internet (e.g `install.packages()` in R or `ssc install ` in Stata) won't work because internet access is restricted in the server.
 
 ## Requesting additional R packages
-The OpenSAFELY R Image ships with a [given set of packages](https://github.com/opensafely/r-docker/blob/master/Dockerfile) baked in.
+The OpenSAFELY R Image ships with a given set of packages baked in. Currently the best way to see the packages that are available is to grab them from a recently-updated docker image as follows:
+
+* Create an R script, `get_packages.R` say, with the following line:
+```R
+write.csv(installed.packages()[, c("Package","Version")], row.names=FALSE, file="output/installed_packages.csv")
+```
+* Add an action to `project.yaml` that runs this script:
+```yaml
+  get_packages:
+    run: r:latest analysis/get_packages.R
+    outputs:    
+      moderately_sensitive:
+        cohort: output/installed_packages.csv
+```
+* Run that action using `opensafely run get_packages`
+
+This will create a CSV file containing the version of available packages in OpenSAFELY's R docker image.  If you don't run this script via `opensafely` then it will return the packages available on your local R installation, not the packages in the docker image.
 
 If the package you need is not available, you can request it as follows:
 
-* Make an Issue in https://github.com/opensafely/r-docker/issues
+* Make a new Issue in [https://github.com/opensafely/r-docker/issues](https://github.com/opensafely/r-docker/issues)
 * Explain the rationale
 * Get another member of the suppoRt group to chip in on the issue (e.g. do they agree; do they think a different package might do the same thing better)
 * Assign to Simon when discussion finished.
```

Comment:
Maybe assign to @developers instead of me? I'll still see it, but it's not dependent on me then.